jerry-core/parser/regexp/re-parser.h

```diff
@@ -78,7 +78,7 @@ typedef enum
 /**
  * Undefined character (out of the range of the codeunit)
  */
-#define RE_CHAR_UNDEF 0xFFFFFFFF
+#define RE_CHAR_UNDEF 0xFFFF
```

Comment:
Let me change my review. This can be correct. According to the Unicode standard (http://www.unicode.org/versions/Unicode5.2.0/ch16.pdf, section 16.7):

> Noncharacters: U+FFFE, U+FFFF, and Others

> U+FFFF and U+10FFFF. These two noncharacter code points have the attribute of being
associated with the largest code unit values  for particular Unicode encoding forms. In
UTF-16, U+FFFF is associated with the largest 16-bit code unit value, FFFF_16. U+10FFFF is associated with the largest legal UTF-32 32-bit code unit value, 10FFFF_16. This attribute
renders these two noncharacter code points useful for internal purposes as sentinels. For
example, they might be used to indicate the end of a list, to represent a value in an index
guaranteed to be higher than any valid character value, and so on.

So, we can use this as a marker. But why use it only here and why not move this definition also to lit-strings.h?